lib/services/nativescript-migration-service.ts

```diff
@@ -0,0 +1,143 @@
+///<reference path="../.d.ts"/>
+"use strict";
+
+import Future = require("fibers/future");
+import path = require("path");
+import helpers = require("./../helpers");
+
+export class NativeScriptMigrationService implements IFrameworkMigrationService {
+	private static TYPESCRIPT_ABBREVIATION = "TS";
+	private static JAVASCRIPT_ABBREVIATION = "JS";	
+	private nativeScriptMigrationFile: string = path.join(__dirname, "../../resources/NativeScript", "nativeScript-migration-data.json");
```

Comment:
You can use `this.$resources.resolvePath("NativeScript")` - this will return the path to NativeScript folder